dojo/templates/base.html

```diff
@@ -325,6 +325,33 @@
                                             </ul>
                                             <!-- /.nav-second-level -->
                                         </li>
+                                        {% if request.user|check_problems_enabled %}
```

Comment:
If possible, we should drop the `request.user` parameter, likely by changing `check_problems_enabled` to be a simple tag instead of a `filter`.

Reply:
It seems that the `simple_tag` cannot return a variable that you can use with if, and the filter only accepts something that can be used if a parameter exists. If it doesn't pass, it's always interpreted as if the function doesn't exist.
I'm doing something similar to the logic in the `can_add_product` function in `navigation_tags`. The function is receiving the parameter but not using it, and it's still working.